composer.json

```diff
@@ -13,16 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": "^7.1"
     },
     "require-dev": {
-        "symfony/framework-bundle": "^2.7||^3.0||^4.0",
-        "symfony/console": "^2.7||^3.0||^4.0",
-        "twig/twig": "^1.26||^2.0",
-        "phpunit/phpunit": "^4.8"
-    },
-    "conflict": {
-        "twig/twig": "<1.26"
+        "symfony/framework-bundle": "^3.4 || ^4.0",
```

Comment:
this should be moved to the require section. I guess it just wasn't there yet because of silex. But silex support has been dropped.

Reply:
You are correct. I've updated this in https://github.com/ekino/EkinoNewRelicBundle/pull/138/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R16